### ISSUE TITLE
cocoeval花的时间变长了

### ISSUE THREAD

**Issue by vvhj** [Nioolek/PPYOLOE_pytorch#13]
这个是我在yolox上测试的时间
2022-01-22 08:28:29.327 | INFO     | yolox.evaluators.coco_evaluator:evaluate_prediction:208 - Evaluate in main process...
2022-01-22 08:28:33.417 | INFO     | yolox.evaluators.coco_evaluator:evaluate_prediction:241 - Loading and preparing results...
2022-01-22 08:28:39.083 | INFO     | yolox.evaluators.coco_evaluator:evaluate_prediction:241 - DONE (t=5.67s)

这个是我用您这测试的时间
2022-06-13 05:28:58.913 | INFO     | yolox.evaluators.coco_evaluator:evaluate_prediction:244 - Evaluate in main process...
2022-06-13 05:31:54.250 | INFO     | yolox.evaluators.coco_evaluator:evaluate_prediction:277 - Loading and preparing results...
2022-06-13 05:33:08.532 | INFO     | yolox.evaluators.coco_evaluator:evaluate_prediction:277 - DONE (t=74.28s)

用时差10倍

**Comment by sdreamforchen** [comment]
请问eval的时候是重参后的架构吗

**Comment by vvhj** [comment]
> 

分析了一下，推理速度是ok的，感觉问题出在forward_eval 里 
return torch.cat(  # 目标是(1,8400,85)
            [
                pred_bboxes,    #
                torch.ones((b, pred_bboxes.shape[1], 1), device=pred_bboxes.device, dtype=pred_bboxes.dtype),
                cls_score_list.permute(0, 2, 1)
            ],
            axis=-1)

torch.ones((b, pred_bboxes.shape[1], 1), device=pred_bboxes.device, dtype=pred_bboxes.dtype),我怀疑是这行导致每个结果都要计算所以在走cocoeval的时候要花较多的时间
